.github/workflows/tests.yml

```
@@ -5,41 +5,56 @@ on:
   pull_request:
 
 jobs:
-  build:
-
+  generate_conda_packd_envs:
+    name: Generate conda-pack'd envs with Python ${{ matrix.python-version }}
     runs-on: ubuntu-latest
-
     strategy:
       matrix: 
-        python-version: [3.7,3.8]
-
+        python-version: [3.7,3.8,3.9]
       fail-fast: false
+    
     steps:
-
-    - uses: actions/checkout@v2
-
-    - name: start Sirepo-Bluesky container
-      run: bash start_docker.sh -d
-
-    - name: is Sirepo-Bluesky running?
-      run: docker ps -a
-
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-
-    - name: Install
-      shell: bash -l {0}
-      run: |
-        set -vxeuo pipefail
-        pip install --upgrade pip wheel
-        pip install .
-        pip install -r requirements-dev.txt
-        pip list
-
-    - name: Test with pytest
-      shell: bash -l {0}
-      run: |
-        set -vxeuo pipefail
-        pytest -s -v deep_beamline_simulation/tests
+      - name: checkout code
+        uses: actions/checkout@v2
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: "conda-pack"
```

Let's name the environment correspondingly (that can also be taken from the GH Actions env vars):
```suggestion
          activate-environment: "deep-beamline-simulation"
```